module/bdev/raid/raid1.c

```diff
@@ -343,6 +429,7 @@ static struct raid_bdev_module g_raid1_module = {
 	.start = raid1_start,
 	.stop = raid1_stop,
 	.submit_rw_request = raid1_submit_rw_request,
+	.submit_null_payload_request = raid1_submit_null_payload_request,
```

Comment:
Who calls `submit_null_payload_request`?

Reply:
`submit_null_payload_request` is called by `bdev_raid.c:raid_bdev_submit_request` when a `SPDK_BDEV_IO_TYPE_FLUSH` operation is submitted. The flow should be the following:

- call fsync over the exported block device
- NVMe kernel driver sends a flush command via NVMe-oF
- the flush command arrives to the NVMe-oF layer of SPDK
- flush is routed to the namespaces added to the NVMe subsystem, in our case to the raid bdev
- bdev layer submit the flush request to the raid layer
- raid layer submit the request to the raid1 layer
- raid1 layer route the flush to all base bdevs